packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -907,15 +935,107 @@ new PrismaClient({
       )
     }
 
-    async $transaction(promises: Array<any>): Promise<any> {
+    /**
+     * @deprecated
+     */
+    async $___transaction(promises: Array<any>): Promise<any> {
```

Comment:
Could we find another name for it maybe? 

_ often means private which is not what it means here I think?

`$toBeDeprecatedTransaction` maybe or anything else?